app/views/schools/advice/heating_control/not_enough_data.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render 'schools/advice/advice_page', school: @school, advice_pages: @advice_pages, advice_page: @advice_page,
+                                         tab: @tab, data_warning: nil do %>
+  <h2>
+    <%= component 'page_nav/collapse_button', display: 'inline' %>
+    <%= t('advice_pages.heating_control.not_enough_data.title') %>
+  </h2>
+  <p>
+    <%= t('advice_pages.heating_control.not_enough_data.p1') %>
+    <ul>
+      <li><%= t('advice_pages.heating_control.not_enough_data.p1_li1') %></li>
+      <li><%= t('advice_pages.heating_control.not_enough_data.p1_li2') %></li>
+      <li><%= t('advice_pages.heating_control.not_enough_data.p1_li3') %></li>
+    </ul>
+  </p>
+  <% if @analysable.data_available_from.present? %>
```

Comment:
it seems like this is always nil so perhaps shouldn't have? https://github.com/Energy-Sparks/energy-sparks/blob/bae329427d699a77fc2521a87c494d4a2da0b1ad/app/services/schools/advice/heating_control_service.rb#L37

Reply:
Sorry, I'd hadn't realised we didn't have an available date here, so yes best remove.

Reply:
Actually, I think its fine to leave here for now. We can sort out estimating a suitable time separately.